libmscore/ambitus.cpp

```diff
@@ -621,7 +621,8 @@ void Ambitus::updateRange()
             segm = segm->nextCR();
             }
 
-      if (pitchTop > -1000) {             // if something has been found, update this
+      if (pitchTop > SCHAR_MIN-1/* || pitchBottom < SCHAR_MAX+1 || tpcTop != Tpc::TPC_INVALID || tpc_Bottom != Tpc::TPC_INVALID*/) {
```

Comment:
Getting rid if the 'magic numbers'. Not sure whether to uncomment that commented part, or whather it matters at all, `tcp_Top` gets change along with `pitchTop`, tpc_Bottom` along with `pitchBottom`, and I don't see how `pitchBottom` could get changed ever get changed but not `pitchTop` or vice versa.